tests/end_to_end/test_accounts_page.py

```diff
@@ -2,50 +2,45 @@
 import os
 import re
 
-import pytest
 from playwright.sync_api import expect
 
 
-@pytest.mark.skip(reason="Not authenticating test users.")
-def test_accounts_page(end_to_end_authenticated_context):
+def _bypass_sign_in(end_to_end_context):
```

Comment:
Noting that this isn't what I had in mind in terms of allowing a user through, but I like that it keeps all of this consolidated with the tests!  If we can get this fully working then it'd be great to keep this as is.

If not, we may have to go back and consider updating the actual authentication decorator or some part of the app to recognize E2E test users and let them through there.

Reply:
fixed